### PR TITLE
Updated addin to check for Military Schema

### PR DESCRIPTION
**Pull request by tlauver**
The addin will now check for the schema rather than the database name.  The first database it detects will be the one it uses that has the correct schema.  It'll check the schema by going through the feature classes and making sure each exist.  If when a user clicks on a style search result, and they don't have the schema, the addin will prompt to install the layer package, after which they can continue in the process.

### CODE REVIEW

File: source/ProSymbolEditor/Models/SchemaDataModel.cs [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ProSymbolEditor
+{
+    public class SchemaDataModel
+    {
+        public bool SchemaExists { get; set; }
```
Comment (csmoore) [inline source/ProSymbolEditor/Models/SchemaDataModel.cs]: Just wondering if all of these properties should be public and set-able

File: source/ProSymbolEditor/ProSymbolEditor.csproj [hunk 2]
```diff
@@ -11522,6 +11523,11 @@
     </Content>
   </ItemGroup>
   <ItemGroup />
+  <ItemGroup>
+    <Content Include="Files\MilitaryOverlay.lpkx">
```
Comment (csmoore) [inline source/ProSymbolEditor/ProSymbolEditor.csproj]: "Files" might be a little too generic (maybe "DeploymentFiles") - but no need to change

File: source/ProSymbolEditor/Models/SchemaDataModel.cs (hunk 1 above)
Comment (tlauver) [inline source/ProSymbolEditor/Models/SchemaDataModel.cs]: FieldsToCheck and FeatureClassExists can be set to private, the other ones are used outside of the model.

File: source/ProSymbolEditor/ProSymbolEditorModule.cs [hunk 3]
```diff
@@ -28,8 +28,7 @@ namespace ProSymbolEditor
     internal class ProSymbolEditorModule : Module
     {
         private static ProSymbolEditorModule _this = null;
-        public static bool _isEnabled = false;
-        public static string WorkspaceString = "militaryoverlay.gdb";
+        public SchemaDataModel DataModel { get; set; }
```
Comment (csmoore) [inline source/ProSymbolEditor/ProSymbolEditorModule.cs]: I worry "SchemaDataModel"/"DataModel" might also be too generic and hard to understand in the code - the official name for this is the ["MilitaryOverlayInformationModel"](http://esriurl.com/MilitaryOverlayInformationModel)  so something like MilitaryOverlaySchema and MilitaryOverlayDataModel might be better

File: source/ProSymbolEditor/Models/SchemaDataModel.cs (hunk 1 above)
Comment (csmoore) [inline source/ProSymbolEditor/Models/SchemaDataModel.cs]: "set" outside the class (setters needed also)?

Comment (tlauver) [inline source/ProSymbolEditor/Models/SchemaDataModel.cs]: Just "get" outside of the model class.  I could set "set" to private.  Want me to make that change?

**Comment by csmoore** [comment]
I really like the new workflow (to prompt to add the MilitaryOverlay.lpkx) - thanks for adding

**Pull request merged**